app/views/errors/other_ways.scala.html

```diff
@@ -28,29 +29,33 @@
 maybeMainDataAttributes = Some(Html("data-journey=marriage-allowance:stage:other-ways")))) {
 
     <header class="page-heading">
-        <h1 class="heading-xlarge">@Html(Messages("technical.other-ways.h1"))</h1>
+        <h1 class="heading-xlarge">@messages("technical.other-ways.h1")</h1>
     </header>
 
-    <p id="pta-redirect">@Html(Messages("technical.other-ways.para0"))</p>
-    <p id="unable-to-confirm-identity">@Html(Messages("technical.other-ways.para1"))</p>
-    <p id="need-nino">@Html(Messages("technical.other-ways.para2"))</p>
-    <p>@Html(Messages("hmrc.contact-details"))</p>
+    <p id="pta-redirect">
+        @messages("technical.other-ways.para0.part1")
```

Comment:
The english message for this has Anchor tag. Needs wrapped in Html

Reply:
Looking at the way this has been split, this looks like a change is needed in the messages file instead. Dont wrap it in HTML

Reply:
fixed in messages